renderer.js

```diff
@@ -16,11 +16,28 @@ setTimeout(() => {
 
 ipcRenderer.on('data', (event, arg) => {
   data = arg;
-  console.log('data received', data)
   voyagerInstance.updateData({
     values: data,
   });
 });
 
 
+let snapshot;
```

Comment:
I was thinking more like saving this to a file for Voyager as if users are using Voyager to explore data and want to continue later.  (Say `.vy` or `.vy.json` and importing back)